classes/class-ep-debug-bar-query-output.php

```diff
@@ -11,6 +11,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
+use ElasticPress\StatusReport\FailedQueries;
```

Comment:
For all these classes we will need to check if they exist before using them, @MARQAS, as the user could still be running EP 4.4.0, for example.